applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -195,11 +197,14 @@ reverse_lookup(JObj) ->
     IP = wh_json:get_first_defined([<<"From-Network-Addr">>
                                     ,<<"Orig-IP">>
                                    ], JObj),
-    case maybe_find_global(IP, Realm) of
+    Port = wh_json:get_first_defined([<<"From-Network-Port">>
```

Comment:
Should put this in a function to `find_port(JObj)` that checks for undefined and casts to integer if not undefined.